internal/channelz/logging.go

```diff
@@ -45,23 +66,48 @@ func Infof(id int64, format string, args ...interface{}) {
 			Severity: CtINFO,
 		})
 	} else {
-		grpclog.InfoDepth(1, msg)
+		logger.InfoDepth(1, msg)
+	}
+}
+
+// InfoToLogger is equivalent to Info except it logs to a specified component logger.
```

Comment:
This is in `internal`.  So we can change `Info`/etc to accept a logger always.  Do we still need the non-logger forms for some uses?  If so, why?

Reply:
You're right, it is internal and the old ones aren't used. I changed it so the with-logger functions replace the old ones.